internal/mining/mining.go

```diff
@@ -40,6 +40,134 @@ const (
 	MinHighPriority = dcrutil.AtomsPerCoin * 144.0 / 250
 )
 
+// Config is a descriptor containing the mining configuration.
+type Config struct {
+	// Policy houses the policy (configuration parameters) which is used to control
+	// the generation of block templates.
+	Policy *Policy
+
+	// TxSource represents a source of transactions to consider for inclusion in
+	// new blocks.
+	TxSource TxSource
+
+	// TimeSource defines a time source to use.
```

Comment:
These comments should be made a bit more explicit like the others as opposed to stating the obvious.  They don't seem to be especially useful as they are.